File: KingdomCodeTool/wwwroot/Download/202503250844/Angular/shared/KhamBenh_SinhHieu.service.ts
import { Injectable } from '@angular/core';
import { HttpClient, HttpParams } from '@angular/common/http';
import { environment } from 'src/environments/environment';
import { KhamBenh_SinhHieu } from './KhamBenh_SinhHieu.model';
import { BaseService } from './Base.service';
@Injectable({
    providedIn: 'root'
})
export class KhamBenh_SinhHieuService extends BaseService{
    DisplayColumns001: string[] = ['Save', 'STT', 'BenhNhan_Id', 'Mach', 'HuyetApThap', 'HuyetApCao', 'NhipTho', 'NhietDo', 'ChieuCao', 'CanNang', 'BMI'];

    List: KhamBenh_SinhHieu[] | undefined;
    ListFilter: KhamBenh_SinhHieu[] | undefined;
    FormData!: KhamBenh_SinhHieu;
    constructor(public httpClient: HttpClient) {
        super(httpClient);
        this.Controller = "KhamBenh_SinhHieu";
    }
}

File: KingdomCodeTool/wwwroot/Download/202503250844/Service/Implement/KhamBenh_SinhHieuService.cs
namespace Service_eHospital_DongNai_A.Implement
{
    public class KhamBenh_SinhHieuService : BaseService<KhamBenh_SinhHieu, IKhamBenh_SinhHieuRepository>
    , IKhamBenh_SinhHieuService
    {
    private readonly IKhamBenh_SinhHieuRepository _KhamBenh_SinhHieuRepository;
    public KhamBenh_SinhHieuService(IKhamBenh_SinhHieuRepository KhamBenh_SinhHieuRepository) : base(KhamBenh_SinhHieuRepository)
    {
    _KhamBenh_SinhHieuRepository = KhamBenh_SinhHieuRepository;
    }
    }
    }

